README.md

```diff
@@ -6,81 +6,74 @@ Note that over time, this job, and all the jobs called by it, will be migrated t
 
-1. Once created you'll see our PRs here:
-    * https://github.com/operator-framework/community-operators/pulls?q=%22Update+eclipse-che+operator%22+is%3Aopen
+2. To start a release, use the [Release - Orchestrate Overall Release Phases] workflow to begin with releasing . In the input, you have to provide the version of Che, DevWorkspace controller, and phases. 
 
-1. If tests fail or community operator PRs get stalled:
-            * Ping @j0zi (Jozef Breza) or @mavala (Martin Vala)
+2.1 If one of the workflows has crashed, inspect it. Apply fixes if needed, and restart it. You can restart individual workflow, or whole phase in orchestration job, whichever is simpler.
+2.2 Keep in mind, that sometimes you'll need to regenerate tags, or skip certain substeps in that job. Also ensure that correct code is in place, whether it is main or bugfix branch.
+2.3 Sometimes, the hotfix changes to the workflow can take too long to get approved and merged. In certain situations, we can use the modified workflow file, which is pushed in its own branch, and then trigger the workflow, while specifying the branch with our modified workflow. 
 
-1. After creating the PRs, add a link to the new PRs from the release issue, eg.,
-    * https://github.com/eclipse/che/issues/18468 -> 
-    * https://github.com/operator-framework/community-operators/pulls?q=is%3Apr+7.23.0
+3. When Che Operator PRs have been generated, you must wait for the approval of PR checks, that are in that repository. If there are any questions, you can forward them to the check maintaners (Deploy team). When PRs are merged, the last batch of projects will be triggered to release
+3.1 Chectl PR has to be closed manually, after they're generated, and all its associated PR checks are passed.
+3.2 Community operator PRs are merged by Operator Framework members, as soon as their tests will pass (in some cases they may require some input from us)
+3.3 Docs PR has to be merged by Docs team.
 
-
-### chectl
-
-TODO: set GH action to trigger when community PRs are merged? Can we trigger across repos? See problem in https://github.com/eclipse/che/issues/18627
-
-
-This depends on the che-operator PRs being merged (but not the community PRs).
-
-1. Run this action: https://github.com/che-incubator/chectl/actions?query=workflow%3A%22Release+-+create+pull+request+for+upcoming+release%22
-
-1. Find the generated PR: https://github.com/che-incubator/chectl/pulls?q=is%3Apr+author%3Aapp%2Fgithub-actions+%22Release+version%22+is%3Aopen
-
-1. Once approved / tests passed / known failures overridden, commit the PR, eg., https://github.com/che-incubator/chectl/pull/1021
-
-1. This second action should run automatically on commit of the above PR: https://github.com/che-incubator/chectl/actions?query=workflow%3A%22Release+-+generate%20release%20and%20push%20artifacts%20to%20github%20pages%20(after%20PR%20approved)%22
+4. If the release is complete, send an e-mail to `che-dev` mailing list, about succesful release.
```

Comment:
should be WHEN the release is complete. Also worth mentioning that the email should remind committers to check for unresolved issues for the current milestone, and to either mark them `closed` or to move them to the next milestone.